rust/cubestore/src/sys/malloc.rs

```diff
@@ -1,3 +1,6 @@
+use crate::util::time_span::warn_long;
+use flatbuffers::bitflags::_core::time::Duration;
```

Comment:
It looks like that IDE create an import wrongly from `flatbuffers` instead of `std::time::Duration`
is it?

Reply:
Yep, thanks.